frontend/src/utils/projectPage.js

```diff
@@ -102,3 +224,12 @@ export const submitKitspaceYaml = async (
   }
   return uploadFile(projectFullname, 'kitspace.yaml', newKitspaceYAML, user, csrf)
 }
+
+/**
+ *
+ * @param {string} assetsPath url for project assets.
+ * @returns {string} url for assets processing status.
+ */
+export const getStatusPathFrom = assetsPath => {
+  return assetsPath.replace(/\/files/, '/status')
+}
```

Comment:
This will fail on a project called "/files...`

Reply:
Resolved in afe0464.

Reply:
The `From` made its usage reads like English, `getStatusPathForm(assestsPath)`. :laughing: